e2e/features/submit-appeal.feature

```diff
@@ -58,6 +55,3 @@ Feature: Submit appeal application
     And I should see the text `Review the appeal application form data and cross reference it with Home Office data. If the appeal looks valid, you must tell the respondent to supply their evidence.`
     And I should see the text `Request respondent evidence`
     And I should see the text `The first column of information is sourced from the appeal application form, the second column of information is from Home Office records`
-    And I should see the text `Appellant details`
-    And I should see the text `Application details`
```

Comment:
These labels should still be there right? are the checks removed because they are inside a table?

Reply:
Correct. E2E cannot see the fields inside table